Resources/Prototypes/_Goobstation/Entities/Objects/Weapons/Guns/Battery/x-00-weapons.yml

```
@@ -22,27 +28,22 @@
     maxCharge: 1000
     startingCharge: 1000
   - type: Gun
-    fireRate: 1.5
+    fireRate: 1
     selectedMode: FullAuto
     availableModes:
     - FullAuto
     - SemiAuto
     soundGunshot:
       path: /Audio/Weapons/Guns/Gunshots/laser3.ogg
     defaultDirection: 1, 0
-  - type: BatterySelfRecharger
-    autoRecharge: true
-    autoRechargeRate: 15
-    autoRechargePause: true
-    autoRechargePauseTime: 20
   - type: HitscanBatteryAmmoProvider
     proto: RedMediumSplitLaser
-    fireCost: 100
+    fireCost: 35
```

im not entirely insane tho
```suggestion
    fireCost: 100